pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -9833,18 +9833,6 @@ final: prev:
     meta.homepage = "https://github.com/rust-lang/rust.vim/";
   };
 
-  rustaceanvim = buildNeovimPlugin {
```

Comment:
The hybrid way made sense while there was interrogations about how well we can support luarocks.org. It made it possible to bypass the nixpkgs lua infrastructure easily. 
Now that the nixpkgs infra is a lot better and we have NURR, I believe we have robust enough bases to have something that would automate the
```
rustaceanvim = neovimUtils.buildNeovimPlugin { luaAttr = "rustaceanvim"; };
```
part.
maybe sthg like `buildNeovimPlugin lua.pkgs.rustaceanvim` with lua coming from neovim. I dont have all the details in mind but something worth investigating. cc @GaetanLepage